src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const tbl = document.getElementsByTagName('table')[0];
+
+for (const node of tbl.children) {
+  for (const n of node.children) {
+    const extra = document.createElement(n.children[0].tagName);
+
+    extra.textContent = n.children[1].textContent;
+
+    const cols = [...[...n.children].slice(0, 4), extra,
+      ...[...n.children].slice(4)];
```

Comment:
it will be better to use `cloneNode` method

Reply:
done